### PR TITLE
Use GitHub CI for demo app

### PR DESCRIPTION
**Pull request by marcus-daily**
Automatically build the app every day + upload the APK to GitHub.

### CODE REVIEW

**Comment by wcarss** [comment]
not that storage is a big concern, but we could likely trigger this on PRs and merges to main instead, and it would produce _far_ fewer artifacts -- I think we have ~1 merge / month 😄

**Comment by wcarss** [comment]
(also, having this build automatically is a great idea 🎉 )

**Comment by marcus-daily** [comment]
@wcarss Thanks!

I opted for building once per day (in addition to PRs/merges) to catch any breakage not caused by us (for example, Gradle repos going down, GitHub runner updates, etc) so they don't show up for the first time during the monthly merge :grin: Happy to go either way though, or dial down the frequency to once per week -- just let me know.

**Comment by wcarss** [comment]
+1, those are reasonable things to have some warning of -- maybe a mix of weekly and on push to main?

**Comment by marcus-daily** [comment]
Done :)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on:
+    push:
```
Comment (wcarss) [inline .github/workflows/ci.yml:4]: ah, interesting -- I was unfamiliar with these being used without values to operate on any push, any PR, etc.

**Review by wcarss — APPROVED** [review]

**Comment by marcus-daily** [comment]
@wcarss Looks like this still needs merging :)

**Pull request merged**